crates/pop-cli/src/commands/new/contract.rs

```diff
@@ -1,20 +1,27 @@
 // SPDX-License-Identifier: GPL-3.0
 
 use crate::style::Theme;
+use anyhow::Result;
 use clap::{
 	builder::{PossibleValue, PossibleValuesParser, TypedValueParser},
 	Args,
 };
 use cliclack::{clear_screen, confirm, input, intro, log::success, outro, outro_cancel, set_theme};
 use console::style;
-use pop_contracts::{create_smart_contract, is_valid_contract_name, Template};
+use pop_contracts::{create_smart_contract, is_valid_contract_name, ContractType, Template};
 use std::{env::current_dir, fs, path::PathBuf, str::FromStr};
 use strum::VariantArray;
 
 #[derive(Args, Clone)]
 pub struct NewContractCommand {
 	#[arg(help = "Name of the contract")]
 	pub(crate) name: Option<String>,
+	#[arg(
+		help = "Contract type.",
+		default_value = ContractType::Examples.as_ref(),
+		value_parser = crate::enum_variants!(ContractType)
+	)]
+	pub(crate) contract_type: Option<ContractType>,
```

Comment:
I received feedback on a pending PR suggesting that we should make the flags shorter. I like it, do you think makes sense to call it only type? https://github.com/r0gue-io/pop-cli/pull/209#discussion_r1650551879

Reply:
Unfortunately, type is a reserved keyword in Rust. Could maybe do `category` instead?

Reply:
Let's keep it contract_type then, c_type is a bit confusing